### PR TITLE
genesys: init at 1.0.7

### PR DESCRIPTION
**Pull request by rogarb**
###### Description of changes

https://github.com/mrlem/genesys/

A family tree generator.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/genesys/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchurl, jre, graphviz, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "genesys";
+  version = "1.0.7";
+
+  src = fetchurl {
+    url = "https://github.com/mrlem/genesys/releases/download/v${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-I1lEVvwRiGf1f4zUtqKhSb+it/nC8WAmw5S6edquOj8=";
```
Comment (drupol) [inline pkgs/applications/misc/genesys/default.nix:9]:
```suggestion
    hash = "sha256-I1lEVvwRiGf1f4zUtqKhSb+it/nC8WAmw5S6edquOj8=";
```

File: pkgs/applications/misc/genesys/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # The package is distributed as a prebuilt JAVA binary
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
```
Comment (drupol) [inline pkgs/applications/misc/genesys/default.nix:18]: Missing hooks.

**Review by drupol — CHANGES_REQUESTED** [review]

**Review by drupol — APPROVED** [review]
LGTM now!

File: pkgs/applications/misc/genesys/default.nix (hunk 2 above)
Comment (rogarb) [reply pkgs/applications/misc/genesys/default.nix:18]: Solved in new version

**Pull request merged**